policy-suggest-kube-system-to-host-pu.yaml

```diff
@@ -30,12 +30,19 @@ data:
             description: "Ruleset automatically created by an Out of Box template."
             propagate: true
             subject:
+            - - "app:host:type=kubernetes"
             - - "@org:kubernetes=kube-system"
             {{- range $_, $orgmeta := .Aporeto.OrganizationalMetadata }}
             {{- if hasPrefix "@org:kubernetes=" $orgmeta }}{{- else }}
               - {{ $orgmeta | quote }}
             {{- end }}
             {{- end }}
+            incomingRules:
+            - action: Allow
+              protocolPorts:
+              - any
```

Comment:
```suggestion
              - "any"
```

Reply:
@ericrpowers I don't see any `protocolPorts` being wrapped in double quotes 🤔  Is this something new?

Reply:
It is wrapped 6 lines below (line 49 in the new file).

Reply:
I'm wondering why this isn't the case across all the recipes, as per `policy-suggest-k8s-api-server.yaml`